recipes/kernel/kmod.yaml

```
@@ -1,29 +1,37 @@
 inherit: [autotools]
 
 metaEnvironment:
-    PKG_VERSION: "22"
+    PKG_VERSION: "33"
 
 privateEnvironment:
     AUTOTOOLS_AUTO_STATIC: "no"
 
 checkoutSCM:
     scm: url
     url: ${KERNEL_MIRROR}/linux/utils/kernel/kmod/kmod-${PKG_VERSION}.tar.gz
-    digestSHA1: "e07a6245a040e8c18bff39b302d6d09bcbdf57b6"
+    digestSHA1: "1514ec6d35a35ba625e2e817e8646c23ee0437e7"
     stripComponents: 1
 
 buildScript: |
-    autotoolsBuild $1
+    autotoolsBuild $1 \
+        --disable-manpages
 
-# We have to create the symlinks ourself. Not sure why "make install" doesn't
-# do it
-packageScript: |
-    autotoolsPackageTgt
-    mkdir -p usr/sbin
+    # We have to create the symlinks ourself. Not sure why "make
+    # install" doesn't do it
+    mkdir -p install/usr/sbin
     for i in depmod insmod lsmod modinfo modprobe rmmod ; do
-        ln -s ../bin/kmod usr/sbin/$i
+        ln -s ../bin/kmod install/usr/sbin/$i
     done
-    ln -s kmod usr/lsmod
+    ln -s kmod install/usr/lsmod
 
-provideTools:
-    kmod: usr/sbin
+multiPackage:
+    "":
+        packageScript: autotoolsPackageTgt
+        provideTools:
+            kmod: usr/sbin
+
+    dev:
+        packageScript: autotoolsPackageDev
+
+    tgt:
+        packageScript: autotoolsPackageTgt
```

I guess these dev/tgt packages are only for libkmod, aren't they? In this case I would like to name them appropriately and only install the libraries in the "lib" target package. But this implies that the shared lib is not in the "" package any more. How about the following? (untested!)

```suggestion
multiPackage:
    "":
        depends:
            - name: kernel::kmod-lib-tgt
              use: []
        packageScript: autotoolsPackageBin
        provideDeps: [ "*-tgt" ]
        provideTools:
            kmod: usr/sbin

    lib-dev:
        packageScript: autotoolsPackageDev

    lib-tgt:
        packageScript: autotoolsPackageLib
```

---

Agreed and done.